src/components/LoginComponent/logincomponent.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../assets/sass/main.scss";
+
+.login-box {
```

Comment:
Maybe will be better if you center this component in the Login Page and not here

Reply:
Thanks. done